src/SConscript

```diff
@@ -12,9 +12,6 @@ HEADERS_GURT = ['dlog.h', 'debug.h', 'common.h', 'hash.h', 'list.h',
                 'heap.h', 'fault_inject.h', 'debug_setup.h',
                 'types.h', 'atomic.h', 'slab.h']
 
-# Keep versioned libs for now to avoid any conflict with 1.0
-CART_VERSION = "4.9.0"
```

Comment:
I don't think you can do this.  (Right or wrong) there are consumers of libgurt outside of DAOS:

Problem 1: package ior-hpc-3.3.0-13.633.g89fd198.el7.x86_64 requires libgurt.so.4()(64bit), but none of the providers can be installed

Reply:
ok, added version back for gurt

Reply:
mpich seems to want cart.  Not sure why. I'm pretty sure it's an indirect dependence